packages/mds-stream/kafka/helpers.ts

```diff
@@ -1,70 +1,70 @@
-import Kafka, { ProducerStream, ConsumerStream } from 'node-rdkafka'
+import { Kafka, Producer, EachMessagePayload, Consumer } from 'kafkajs'
 import log from '@mds-core/mds-logger'
-import {
-  ProducerOptions,
-  ProducerStreamOptions,
-  Producer,
-  Consumer,
-  ConsumerOptions,
-  ConsumerStreamOptions
-} from './types'
+import { v4 as uuid } from 'uuid'
 
 const {
   env: { KAFKA_HOST = 'localhost:9092' }
 } = process
 
-export const defaultKafkaErrorHandler = async (err: object) => {
-  await log.error(`Kafka Error ${JSON.stringify(err)}`)
-}
+export const createWriteStreamWrapper = async () => {
+  try {
+    const kafka = new Kafka({
+      clientId: `writer-${uuid()}`,
+      brokers: [KAFKA_HOST]
+    })
 
-export const createWriteStreamWrapper = (
-  producerOptions: Partial<ProducerOptions>,
-  streamOptions: Partial<ProducerStreamOptions>,
-  errorHandler?: (err: any) => Promise<void>
-) => {
-  const stream = ((Kafka.Producer as unknown) as Producer).createWriteStream(
-    { ...producerOptions, 'metadata.broker.list': KAFKA_HOST, 'queue.buffering.max.messages': 100000 },
-    {},
-    { ...streamOptions }
-  )
+    const producer = kafka.producer()
 
-  stream.on('error', errorHandler ?? defaultKafkaErrorHandler)
-  return stream
+    await producer.connect()
+    return producer
+  } catch (err) {
+    await log.error(err)
+  }
 }
 
-export const createReadStreamWrapper = (
-  consumerOptions: Partial<ConsumerOptions>,
-  streamOptions: ConsumerStreamOptions,
-  readCb: (message: Kafka.ConsumerStreamMessage) => void,
-  errorHandler?: (err: any) => Promise<void>
+export const createReadStreamWrapper = async (
+  topic: string,
+  readCb: (message: EachMessagePayload) => Promise<void>
 ) => {
-  const stream = ((Kafka.KafkaConsumer as unknown) as Consumer).createReadStream(
-    { ...consumerOptions, 'metadata.broker.list': KAFKA_HOST, 'group.id': 'default' },
-    {},
-    { ...streamOptions }
-  )
+  try {
+    const kafka = new Kafka({
+      clientId: `reader-${uuid()}`,
+      brokers: [KAFKA_HOST]
+    })
+
+    const consumer = kafka.consumer({ groupId: 'test-group' })
 
-  stream.on('error', errorHandler ?? defaultKafkaErrorHandler)
-  stream.on('data', readCb)
-  return stream
+    await consumer.connect()
+    await consumer.subscribe({ topic, fromBeginning: true })
```

Comment:
Wondering if we really want `fromBeginning` to be always true?

Reply:
I don't think we do -- ideally it'd be nice to have the consumers:
* Assume a persistent name (somehow)
* If they left off somewhere previously, start back where they were before
* Otherwise, start reading new (not from beginning), overridable by a param

Reply:
Probably want to support passing in a `clientId` value and maybe abstract some of the subscription options.